src/main/scala/com/yotpo/metorikku/configuration/metric/Output.scala

```diff
@@ -6,6 +6,7 @@ import com.fasterxml.jackson.module.scala.JsonScalaEnumeration
 case class Output(name: Option[String],
                   dataFrameName: String,
                   @JsonScalaEnumeration(classOf[OutputTypeReference]) outputType: OutputType.OutputType,
+                  reportLag: Option[String],
```

Comment:
It's not the same name you use inside... you need 2:
reportLag - boolean
reportLagTimeColumn - string